manifests/init.pp

```diff
@@ -32,6 +32,10 @@
 #                                   Setting this to anything non-empty causes
 #                                   the apache vhost to set up a proxy for all
 #                                   certificates pointing to the value.
+# $parent_reverse_proxy::           Add reverse proxy to the parent
```

Comment:
@stbenjam made some effort to integrate templating with the reverse proxy. I'll close those PRs for now.